tools/train.py

```diff
@@ -128,8 +128,9 @@ def run(FLAGS, cfg, place):
     optimizer = create('OptimizerBuilder')(lr, model.parameters())
 
     # Init Model & Optimzer   
+    start_epoch = 0
     if FLAGS.weight_type == 'resume':
-        load_weight(model, cfg.pretrain_weights, optimizer)
+        start_epoch = load_weight(model, cfg.pretrain_weights, optimizer)
```

Comment:
load_weight is used in tools/infer.py and tools/eval.py as well

Reply:
Done, thanks!